fe1-web/__tests__/network/MessageApi.test.ts

```diff
@@ -85,8 +85,8 @@ function checkDataCreateLao(obj: MessageData) {
   expect(data.name).toBeString();
   expect(data.name).toBe(mockEventName);
 
-  expect(data.creation).toBeNumber();
-  expect(data.creation).toBeGreaterThan(0);
+  expect(data.creation).toBeNumberObject();
+  expect(data.creation.valueOf()).toBeGreaterThan(0);
```

Comment:
I wonder how it was working before! 🤔

Reply:
It was working because the mock data was in the 'wrong' type :)